source/gfoidl.Stochastics/Statistics/Sample.AddRange.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using gfoidl.Stochastics.Builders;
+
+namespace gfoidl.Stochastics.Statistics
+{
+    partial class Sample
+    {
+        public IEnumerable<double> AddRange(IEnumerable<double> values)
```

Comment:
So `AddRange` is lazy (due `IEnumerable` return). I think it would be more obvious if this methods iterates over the `values` instantely, and for the fluent-interface case provide another metho like 
```c#
public void AddRange(IEnumerable<double> values);
public IEnumerable<double> AddRangeWithIteration(IEnumerable<double> values);
```
or similar.

Reply:
Done. PR updated.